bpf-oci/src/lib.rs

```diff
@@ -0,0 +1,99 @@
+//!  SPDX-License-Identifier: MIT
```

Comment:
separate the bpf-oci to another repo?

Reply:
Done now. It could be found at https://github.com/eunomia-bpf/bpf-oci